doc/user/content/sql/patterns/window-functions.md

```diff
@@ -1,8 +1,126 @@
 ---
-title: "Window functions"
-description: "Use lateral joins as a streaming-efficient stand-in for many window functions."
-draft: true
+title: "Window functions (OVER clause)"
+description: "Avoid window functions, e.g., use lateral joins as a streaming-efficient stand-in for many window functions."
 menu:
   main:
     parent: 'sql-patterns'
 ---
+
+Materialize currently does not handle window functions (`OVER` clause) efficiently for large partition sizes: when an input record is added/removed/changed, the system recomputes the results for the entire partition that the changed record belongs to. We are actively working on making window functions more efficient in Materialize, but for the time being, this page shows some workarounds for avoiding the use of window functions.
```

Comment:
I would leave out "We're actively working on" and replace by a less committal statement. We're planning it at the moment.

Reply:
Changed to "We are planning to make window functions more efficient in Materialize"